src/app/qrcode/page.tsx

```diff
@@ -0,0 +1,70 @@
+'use client';
+
+import Image from 'next/image';
+import { useState, useEffect } from 'react';
+
+import CategoryItem from '@/components/category-item';
+import Modal from '@/components/modal-shorten';
+import QrHomeButton from '@/components/qr-home-button';
+import SelectInput from '@/components/select-input';
+import ShortenTools from '@/components/shorten-tools';
+import TextInput from '@/components/text-input';
+import meService from '@/services/me.service';
+import { tools } from '@/services/tool.service';
+import urlService from '@/services/url.service';
+import Organization from '@/types/organization-type';
+import Url from '@/types/url-type';
+
+export default function qrHomePage() {
```

Comment:
UPPERCASE on first character of component 
MUST BE QRHomePage
```suggestion
export default function QRHomePage() {
```